freeipa/src/main/java/com/sequenceiq/freeipa/service/image/userdata/UserDataService.java

```diff
@@ -79,6 +79,8 @@ public void regenerateUserData(Long stackId) {
     public void updateJumpgateFlagOnly(Long stackId) {
         LOGGER.debug("Updating Jumpgate flag in user data for stack {}", stackId);
         ImageEntity image = imageService.getByStackId(stackId);
+        image.setUserdata(image.getUserdata().replace("IS_CCM_ENABLED=true", "IS_CCM_ENABLED=false"));
```

Comment:
please make it similar to the one in core service (replace chained)

Reply:
ok